lib/api/objectPutCopyPart.js

```diff
@@ -22,21 +26,24 @@ import validateHeaders from '../utilities/validateHeaders';
  *                            includes normalized headers
  * @param {string} sourceBucket - name of source bucket for object copy
  * @param {string} sourceObject - name of source object for object copy
+ * @param {string} reqVersionId - versionId of the source object for copy
  * @param {object} log - the request logger
  * @param {function} callback - final callback to call with the result
  * @return {undefined}
  */
 export default
 function objectPutCopyPart(authInfo, request, sourceBucket,
-    sourceObject, log, callback) {
+    sourceObject, reqVersionId, log, callback) {
     log.debug('processing request', { method: 'objectPutCopyPart' });
     const destBucketName = request.bucketName;
     const destObjectKey = request.objectKey;
     const mpuBucketName = `${constants.mpuBucketPrefix}${destBucketName}`;
+    let sourceVerId = undefined;
```

Comment:
please don't define this out here.  instead pass it along in the waterfall once you have the value.